_ss_environment.php.default

```diff
@@ -0,0 +1,13 @@
+<?php
+
+define("SS_ENVIRONMENT_TYPE", "dev");
+define("SS_DEFAULT_ADMIN_USERNAME", "admin");
+define("SS_DEFAULT_ADMIN_PASSWORD", "password");
+define("SS_DATABASE_SERVER", "localhost");
+define("SS_DATABASE_USERNAME", "root");
+define("SS_DATABASE_PASSWORD", "root");
+define("SS_DATABASE_NAME", "SS_mysite");
+define("ELASTICSEARCH_HOST", "localhost");
```

Comment:
do we need ELASTICSEARCH_HOST etc?  most people aren't going to use these

Reply:
It's required if you want to use search in addons

Reply:
ah, lol, though you were doing a PR for a default Vagrantfile, not something for addons.  ignore this :-)